versions/build.s4.ver

```diff
@@ -0,0 +1,33 @@
+export hpc_ver=1.1.0
```

Comment:
`hpc_ver` should be incremented to 1.2.0.  That's my bad for not doing this sooner.  Would you mind bumping it up?

Reply:
Updated @ f1322477.